src/pages/SurveyList/index.jsx

```diff
@@ -1,70 +1,47 @@
-import style from "./style.module.css";
-import { Table } from "antd";
-
-const dataSource = [
-  {
-    id: 1,
-    name: "PCL-5",
-    questionsAmount: 5,
-    date: "16-01-2021",
-  },
-  {
-    id: 2,
-    name: "GAD",
-    questionsAmount: 20,
-    date: "01-06-2021",
-  },
-  {
-    id: 3,
-    name: "PST-420-BLAZEIT",
-    questionsAmount: 200,
-    date: "01-01-2022",
-  },
-  {
-    id: 4,
-    name: "THC",
-    questionsAmount: 55,
-    date: "01-01-2022",
-  },
-];
+import style from './style.module.css'
+import { Table } from 'antd'
+import { useEffect, useState } from 'react'
+import { getSurveysList } from '../../utils/api'
 
 const columns = [
   {
-    title: "id",
-    dataIndex: "id",
-    key: "id",
+    title: 'id',
+    dataIndex: 'id',
+    key: 'id',
   },
   {
-    title: "Name",
-    dataIndex: "name",
-    key: "name",
+    title: 'Name',
+    dataIndex: 'name',
+    key: 'name',
   },
   {
-    title: "questionsAmount",
-    dataIndex: "questionsAmount",
-    key: "questionsAmount",
+    title: 'questionsAmount',
+    dataIndex: 'questionsAmount',
+    key: 'questionsAmount',
   },
   {
-    title: "date",
-    dataIndex: "date",
-    key: "date",
+    title: 'date',
+    dataIndex: 'date',
+    key: 'date',
   },
-];
+]
 
-function SurveyList(props) {
+function SurveyList() {
+  const [surveyList, setSurveyList] = useState([])
+  useEffect(() => {
+    getSurveysList().then((surveys) => setSurveyList(surveys))
```

Comment:
add a `.catch` and handle error (you can use this it's pretty cool https://ant.design/components/message/)